Models/data_cleansing.py

```diff
@@ -87,14 +87,22 @@ def _create_dummies_for_categorical_features(design_matrix):
     return design_matrix
 
 
-def clean_data(design_matrix):
+def clean_data(design_matrix, remove_outliers=False):
     """Cleaning raw data before model processing."""
     design_matrix = _impute_missing_values(design_matrix)
     # design_matrix = _normalize_continuous_features(design_matrix)
     design_matrix = _create_dummies_for_categorical_features(design_matrix)
+    if remove_outliers:
+        design_matrix = outlier_treatment(design_matrix)
     return design_matrix
 
 
+def outlier_treatment(design_matrix, m=2.75):
```

Comment:
I was checking how many rows are getting removed by this treatment. 
**m = 3:** 22 rows
**m = 2.5:** 40 rows
40 appeared a bit high to me and 22 a bit low, so I chose an in-between value. Please let me know if I should change it.